test/functionalTest/cases/1316_create_subscription/post_subs_expr_q_nocache.test

```diff
@@ -0,0 +1,253 @@
+# Copyright 2016 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+POST Subscriptions V2 q expression without cache
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB 0 IPv4 -noCache
+accumulatorStart
+
+--SHELL--
+
+#
+# 01. Create subscription with q: temperature==20..40
+# 02. Create entity with t=10 (not notif)
+# 03. Update entity to t=15 (not notif)
+# 04. Update entity to t=25 (notif)
+# 05. Update entity to t=35 (notif)
+# 06. Update entity to t=45 (not notif)
```

Comment:
Might be interesting to test corner cases 20 and 40 here.
If something ever goes wrong here, the probability of failure is always higher in the corner cases